plone/app/z3cform/widgets.zcml

```diff
@@ -223,7 +223,7 @@
 
   <!-- relateditems -->
 
-  <adapter
+  <!-- <adapter
```

Comment:
If we fully switch over to `pat-contentbrowser` without any support for `pat-relateditems` anymore, this can be removed, I guess.

Reply:
Yes ... there are some leftovers for now for testing. I've figure out, that these zcml definitions are hardly ever used, because we always define the `RelatedItemsFieldWidget` with `plone.directives` explicitely in the code ... though there might be some corner cases in addons, where these zcml definitions did their job.